aztec-up/bin/docker-compose.sandbox.yml

```diff
@@ -1,19 +1,15 @@
 services:
   ethereum:
-    image: ghcr.io/foundry-rs/foundry@sha256:29ba6e34379e79c342ec02d437beb7929c9e254261e8032b17e187be71a2609f
-    command: >
-      '
-      [ -n "$$FORK_URL" ] && ARGS="$$ARGS --fork-url $$FORK_URL";
-      [ -n "$$FORK_BLOCK_NUMBER" ] && ARGS="$$ARGS --fork-block-number $$FORK_BLOCK_NUMBER";
-      echo anvil -p $$ANVIL_PORT --host 0.0.0.0 --chain-id 31337 --silent $$ARGS;
-      anvil -p $$ANVIL_PORT --host 0.0.0.0 --chain-id 31337 --silent $$ARGS
-      '
+    image: aztecprotocol/foundry:25f24e677a6a32a62512ad4f561995589ac2c7dc-${ARCH_TAG:-amd64}
+    entrypoint: >
+      sh -c '
+      if [ -n "$FORK_BLOCK_NUMBER" ] && [ -n "$FORK_URL" ]; then
+        exec anvil --silent -p 8545 --host 0.0.0.0 --chain-id 31337 --fork-url "$FORK_URL" --fork-block-number "$FORK_BLOCK_NUMBER"
+      else
+        exec anvil --silent -p 8545 --host 0.0.0.0 --chain-id 31337
+      fi'
     ports:
-      - "${ANVIL_PORT:-8545}:${ANVIL_PORT:-8545}"
-    environment:
-      FORK_URL:
-      FORK_BLOCK_NUMBER:
-      ANVIL_PORT: ${ANVIL_PORT:-8545}
+      - 8545:8545
```

Comment:
would this fall under breaking change?

Reply:
Restored, now docker compose files that exposed the envs still do, internal ones that were never configurable in the first place call anvil directly with default values